akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -137,6 +138,32 @@ object GraphStageLogic {
   private object DoNothing extends (() ⇒ Unit) {
     def apply(): Unit = ()
   }
+
+  /**
+   * Minimal actor to work with other actors and watch them in a synchronous ways
+   */
+  class StageActorRef(asyncCallback: AsyncCallback[(ActorRef, Any)], override val path: ActorPath) extends akka.actor.MinimalActorRef {
+    override def provider: ActorRefProvider =
+      throw new UnsupportedOperationException("GraphStage MinimalActor does not provide")
```

Comment:
Because it is reluctant to do so. Grumpy ActorRef it is. We should convince it though.